docs/faq/glossary.md

```diff
@@ -80,6 +81,11 @@ so-called [partner plugins](#partner-plugin).
 The backbone of the Camino Messenger infrastructure. Essentially, it's a server
 relaying messages between bots. Currently, Matrix messenger server is used.
 
+## Camino Messenger Account
+
+The Camino Messenger Account is a smart contract (actually an ensemble of smart contracts), acting as the financial backbone of the [Partner Configuration](#partner-configuration), holding the funds designated for trading on behalf of the Partner on the Camino Messenger, and paying / receiving payment amounts for bookings.  
+It also stores all configuration settings such as the services offered and wanted (and their versions), the applicable service fees and the accepted currencies. It also maintains a list of the wallets of the off-chain bots with which the Partner's systems interact with the Messenger.
+
```

Comment:
The Camino Messenger Account can be created from the Partner Showroom of the [Camino Suite](https://suite.camino.network/partners) or by using a [script](https://github.com/chain4travel/camino-messenger-bot?tab=readme-ov-file#camino-messenger-account-cm-account)

More details can be found in our functional documentation: [Camino Partner Showroom](https://docs.camino.network/partners/partner-showroom "Camino Partner Showroom")

Reply:
@SamJaarsma 
@havan 

I would avoid mentioning the script if it is only meant/encouraged to be used for development purposes (test accounts). Or we mention that here. 

If it's ok that live account are created via hardhat, we can keep it but switch the link to c4t branch once the dev is merged.

Reply:
Agreed that live accounts should not be used for Camino, only for development on Columbus. Updated the suggestion to include that

Reply:
done

Reply:
done